### PR TITLE
fix(scope): fix export types & main entry

### PR DESCRIPTION
**Pull request by kwaa**
Part of https://github.com/unocss/unocss/issues/3675

### CODE REVIEW

**Comment by kwaa** [comment]
It looks like `@unocss/nuxt` doesn't output `index.d.cts`.

**Comment by userquin** [comment]
the cjs file is on root

**Comment by kwaa** [comment]
> the cjs file is on root

I know, I just don't quite understand if an `./dist/index.d.ts` declaration file should be used in this case.

**Comment by userquin** [comment]
the cjs file is on root: may we should move it to ESM first with dual ESM/CJS (VueUse  using that approach: you will need to patch generated cjs file)

**Comment by kwaa** [comment]
> the cjs file is on root: may we should move it to ESM first with dual ESM/CJS (VueUse using that approach: you will need to patch generated cjs file)

Hmm... Maybe this should be done in a separate PR.

File: packages/scope/package.json [hunk 1]
```diff
@@ -1,22 +1,28 @@
...
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/unocss/unocss",
+    "directory": "packages/scope"
+  },
+  "exports": {
+    ".": {
+      "import": {
+        "types": "./index.d.ts",
```
Comment (userquin) [inline packages/scope/package.json:16]: shoud be d.mts

Comment (userquin) [reply packages/scope/package.json:16]: upps, there is no d.mts file...

Comment (kwaa) [reply packages/scope/package.json:16]: > from my review this should be ESM first, still missing node16-esm and bundler in your local? can you upload an screenshot?

These two PRs I created via github.com...

Comment (userquin) [reply packages/scope/package.json:16]: ok, I'll try to review it later on my local

**Review by userquin — APPROVED** [review]

File: packages/scope/package.json (hunk 1 above)
Comment (userquin) [reply packages/scope/package.json:16]: imagen

**Pull request merged**